de_web_plugin.cpp

```diff
@@ -2957,7 +2989,9 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const SensorFi
     item = sensorNode.item(RConfigReachable);
     item->setValue(true);
 
-    if (node->isEndDevice())
+    // @manup, will this work?
```

Comment:
For end-devices with power configuration cluster yes. Some battery powered devices don't have them, like Busch-Jaeger, JUNG and dresden-elektronik switches, these only provide battery information via classic PowerDescriptor.

Anyway it's ok to do it this way and maybe to extend it with a NodeDescriptor.rxOnWhenIdle check.

Reply:
Cool, thanks.